### PR TITLE
completes get api/articles endpoint

### PR DESCRIPTION
**Pull request by kevinzehner**
completed the endpoint and tested for correct results and sorting

### CODE REVIEW

File: models/model.js [hunk 1]
```diff
@@ -11,7 +11,14 @@ exports.selectArticleByID = (article_id) => {
       if (rows.length === 0) {
         return Promise.reject({ status: 404, msg: "article does not exist" });
       }
-      console.log(rows);
       return rows[0];
     });
 };
+
+exports.selectArticles = () => {
+  return db.query(`SELECT articles.*, COUNT(comments.comment_id) AS comment_count
```
Comment (saimakhanom) [inline models/model.js]: from here you should only send back the rows property and not the whole response object that comes back

File: __tests__/app.test.js [hunk 2]
```diff
@@ -76,3 +78,37 @@ describe("GET /api/articles/:article_id", () => {
...
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: Check the returned results keys are correct", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        res.body.articles.rows.forEach((article) => {
```
Comment (saimakhanom) [inline __tests__/app.test.js]: this test will pass if 0 articles are returned as if there are 0 elements in the array, the callback fn inside the forEach will not be invoked and the test will pass by default. you'll need to check that the array length is what you expect it to be first, and that way you know the array is not empty

File: models/model.js (hunk 1 above)
Comment (saimakhanom) [reply models/model.js]: Remember this will also impact how you access the data in your tests

**Review by aloush-dev — APPROVED** [review]
Nice job implementing feedback.
Merge when ready.

**Pull request merged**